python/peak_routing_png_image.ipynb

```diff
@@ -0,0 +1,555 @@
+    "\n",
+    "### Requirements:\n",
+    "- Firefox browser (for Chrome uncomment the marked lines. For other supported browsers read the\n",
+    "[webdriver-manager documentation](https://pypi.org/project/webdriver-manager/))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4dc9d48a",
+   "metadata": {},
+   "source": [
+    "#### Import dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "7d4f561f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import os\n",
+    "import time\n",
+    "\n",
+    "import folium\n",
+    "import openrouteservice\n",
+    "import overpy\n",
+    "from PIL import Image\n",
+    "from selenium import webdriver\n",
+    "from webdriver_manager.firefox import GeckoDriverManager\n",
+    "\n",
+    "# from webdriver_manager.chrome import ChromeDriverManager  # uncomment for Chrome"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a62f02e5",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "#### Create an output folder for the maps if it doesn't exist already"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "b199de8b",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "map_folder = f\"{os.getcwd()}/maps\"\n",
+    "\n",
+    "if not os.path.exists(map_folder):\n",
+    "    print(\"Creating maps folder\")\n",
+    "    os.makedirs(map_folder)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cd6218f4",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "#### Get the coordinates of the peaks\n",
+    "You can directly download the peak data via the overpy package.\n",
+    "It uses the same query language as [overpass-turbo](https://overpass-turbo.eu/#).\n",
+    "\n",
+    "Here the peaks where the name matches one of the `peak_names` will be extracted from the given bounding box."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "552fc49c",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Overpass query:\n",
+      "node[\"natural\"=\"peak\"][name~\"^(Zeisigstein|Herkuleskopf|Kristin Hrádek)$\"](50.82263785103416,13.996753692626951,50.843670947425615,14.116744995117186);out;\n",
+      "\n",
+      "Response:\n",
+      "[\n",
+      "  {\n",
+      "    \"name\": \"Herkuleskopf\",\n",
+      "    \"LonLat\": [\n",
+      "      14.0429261,\n",
+      "      50.8422103\n",
+      "    ]\n",
+      "  },\n",
+      "  {\n",
+      "    \"name\": \"Zeisigstein\",\n",
+      "    \"LonLat\": [\n",
+      "      13.9996773,\n",
+      "      50.8241813\n",
+      "    ]\n",
+      "  },\n",
+      "  {\n",
+      "    \"name\": \"Kristin Hr\\u00e1dek\",\n",
```

Comment:
Just noted this weird encoding in the output - is this fixable?

Reply:
indeed -> done